.github/workflows/ci.yml

```diff
@@ -19,6 +19,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - &stable_rust
```

Comment:
nooooo!  no magical YAML please!  esp because github might not support some of it.  KISS!  Copy pasting a single line is OK, and makes it readable